airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ad_report.json

```diff
@@ -5,6 +5,9 @@
     "ad_group_ad.ad.legacy_responsive_display_ad.accent_color": {
       "type": ["null", "string"]
     },
+    "ad_group.id": {
+      "type": ["null", "string"]
```

Comment:
@schlattk type must be "integer", not "string".

Reply:
@augan-rymkhan

Reply:
@augan-rymkhan let me know if any further changes are needed